modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/DiagramBuilder.js

```diff
@@ -335,6 +345,31 @@ export default function DiagramBuilder() {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [currentEditor, deserialize, version]);
 
+	useEffect(() => {
+		let languageId = defaultLanguageId;
+		if (selectedLanguageId) {
+			languageId = selectedLanguageId;
+			const emptyLabel = elements.find((elements) =>
+				findEmptyElements(elements, selectedLanguageId)
+			);
+			if (!emptyLabel) {
+				setTranslations((previous) => {
+					return {...previous, [languageId]: definitionTitle};
+				});
+			}
+			else {
+				delete translations[languageId];
+				setTranslations(translations);
```

Comment:
@M-IgorFranca Please, add a blank line between the update on translations and its usage.

Reply:
Off course. 👍🏾